src/composables/multisigAccounts.ts

```diff
@@ -164,6 +164,7 @@ export function useMultisigAccounts({ store }: UseMultisigAccountsOptions) {
         }
         consensus.expiration_height = Number(consensus.expiration_height);
         consensus.confirmations_required = Number(consensus.confirmations_required);
+        consensus.totalConfirmations = Number(consensus.confirmed_by.length);
```

Comment:
camelCase and snake_case are mixed here. MAybe there is a solution to have one approach here?

Reply:
There was a solution to use `camelcaseKeysDeep` method but it is breaking the array and bigint values. Hence at the moment we have to keep the same